src/main.py

```diff
@@ -63,10 +64,15 @@
 cc.set_protected_keywords(list(FUNC_DICT.keys()))
 
 # Template for SMAPI log info messages
-smapi_log_message_template = string.Template(
+smapi_log_message_template_nofixes = string.Template(
     "**Log Info:** SMAPI $SMAPI_ver with SDV $StardewVersion on $OS, "
     "with $SMAPIMods C# mods and $ContentPacks content packs."
 )
+smapi_log_message_template_fixes  = string.Template(
+    "**Log Info:** SMAPI $SMAPI_ver with SDV $StardewVersion on $OS, "
+    "with $SMAPIMods C# mods and $ContentPacks content packs. \n"
+    "Suggested fixes: $suggested_fixes"
```

Comment:
Just checking - If you have several $suggested_fixes, will it display it as `[fix1, fix2, fix3]` or will it format it to be better English?

Reply:
it seems to be an array, will resolve that